mariadb-init.sql

```diff
@@ -0,0 +1,2 @@
+DROP DATABASE QHAnaExperiments;
```

Comment:
Is this intentional for the docker-compose file? If so it should probably be mentioned in the readme.